02_bubble/SandhyaSMani/02_bubble.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, bubble([]int{3, 4, 1, 5, 2}))
+}
+
+func bubble(s []int) []int {
+	for i := 1; i < len(s); i++ {
```

Comment:
`s` is modified in your implementation. Copy `s` and modify the copy. Test that `s` stays the same after `bubble(s)`